Templates/Specimens_Metadata_2021Q1_template.tsv

```diff
@@ -0,0 +1,18 @@
+Column Name	Description	Ontology identifier	Ontology term name	Ontology Description
+Specimen ID	Unique identifier for this specimen	OBI:0100051	specimen	A material entity that has the specimen role.
+Parent specimen ID	Identifier for this specimen’s parent specimen (optional)	OBI:0100051	specimen	A material entity that has the specimen role.
+Number of subspecimens	A count of specimens derived from this specimen (cumulative)	APOLLO_SV:00000032	count	A measurement datum that is the output of counting.
+Specimen type	Type of specimen (excised tissue block, slice, cell, etc.)	AIO:specimen type name	specimen type name	A textual entity that denotes the type of a specimen.
+Hemisphere	Hemisphere that this specimen came from (left, right, both, unknown)	PATO:0000140	position	A spatial quality inhering in a bearer by virtue of the bearer's spatial location relative to other objects in the vicinity.
+Anatomical structure	Specimens location within anatomical structures from brain atlas	PATO:0000141	structure	A morphology quality inhering in a bearer by virtue of the bearer's relative position, shape, arrangements and connectivity of an organism's various parts; the pattern underlying its form.
```

Comment:
Potential conflations mentioned in the discussion.
- organ
- tissue type
- coordinates

Reply:
For coordinates and non-uberon terms, you may also want a field like `anatomical atlas`.

Reply:
I included a column specifying what anatomical atlas the anatomical structure is defined in. Ideally we would only be working with a structure or list of structures within one atlas, otherwise things could quickly get quite complicated.

Reply:
There should be a `1:1` mapping between structures and the operational definition for their delineation (structure + atlas), so I think this is ok. As I understand it, you can specify a different atlas for each structure, which is the 99% use case.